src/GithubMergeTool/GithubMergeTool.cs

```diff
@@ -52,12 +54,13 @@ public class GithubMergeTool
             string destBranch)
         {
             string prTitle = $"Merge {srcBranch} to {destBranch}";
+            string prBranchName = $"merges/{srcBranch}-to-{destBranch}";
 
             // Check to see if there's already a PR
             // TODO: handle pagination
             // This handles the 100 most recent PRs, which should work for now
             HttpResponseMessage prsResponse = await _client.GetAsync(
-                $"repos/{repoOwner}/{repoName}/pulls?state=open&base={destBranch}&per_page=100");
+                $"repos/{repoOwner}/{repoName}/pulls?state=open&base={destBranch}&per_page=100&head={_username}:{prBranchName}");
```

Comment:
❔  With the additional filters, can you limit this to a single item.

Reply:
I could, but don't see the benefit.

Reply:
I will, however, remove the comment. That is confusing.

Reply:
Yeah, I'm removing the explicit 100 count request and the comment. That seems like the simplest, most straightforward API call, and the logic on the bot side seems simple as well.